README.md

```diff
@@ -1,2 +1,52 @@
 # ProgressionSystem
-Progression System Component for the Bomber project JanSeliv/Bomber
+
+## Overview
+
+It's a Progression System that allows unlocking new characters by playing the game developed for (Bomber project)[https://github.com/JanSeliv/Bomber).
```

Comment:
`(Bomber project` phas wrong link syntax here, so it displays wrong (as text, but not link), is should be as `[name](link)`:
`[Bomber project](https://github.com/JanSeliv/Bomber)`.

Reply:
done